chapters/ch44.md

```diff
@@ -1,85 +1,71 @@
-
+Trong trường hợp này, thuật toán học tập của bạn đã đưa S<sup>x</sup> một điểm chính xác cao hơn S​<sub>out</sub>.​ Tuy nhiên, thuật toán tìm kiếm gần đúng của chúng tôi đã chọn S​<sub>out</sub> thay vì S<sup>x</sup>. Điều này cho bạn biết rằng thuật toán tìm kiếm gần đúng của bạn không chọn giá trị S tối đa hóa Điểm​<sub>A​</sub>(​*S*). Trong trường hợp này, kiểm tra xác minh tối ưu hóa cho bạn biết rằng bạn có vấn đề về thuật toán tìm kiếm và nên tập trung vào đó. Ví dụ: bạn có thể thử tăng độ rộng chùm tia của tìm kiếm chùm tia.
 
 > Case 2: Score​<sub>A​</sub>(​*S*<sup>x</sup>) ≤ Score​<sub>A​</sub>(​*S*<sub>out</sub>)
 
---> _replace THIS LINE by your translation for the above line_
-
+Trường hợp 2: Điểm ​<sub>A​</sub>(​*S*<sup>x</sup>) ≤ Điểm ​<sub>A​</sub>(​*S*<sub>out</sub>)
 
 > In this case, you know that the way you’re computing Score​<sub>A​</sub>(.) is at fault: It is failing to give a strictly higher score to the correct output ​*S*<sup>x</sup> than the incorrect ​S​<sub>out</sub>. The Optimization Verification test tells you that you have an objective (scoring) function problem. Thus, you should focus on improving how you learn or approximate Score​<sub>A​</sub>(​*S*) for different sentences ​*S​*.
 
---> _replace THIS LINE by your translation for the above line_
-
+Trong trường hợp này, bạn biết cách bạn tính toán Điểm​<sub>A​</sub>(.) có lỗi: Không thể cho điểm cao hơn hoàn toàn cho đầu ra chính xác ​*S*<sup>x</sup> so với ​S​<sub>out</sub> không chính xác. Kiểm tra xác minh tối ưu hóa cho bạn biết rằng bạn có vấn đề về hàm tính điểm mục tiêu. Vì vậy, bạn nên tập trung vào việc cải thiện cách bạn học hoặc tính gần đúng Điểm​<sub>A​</sub>(​*S*) cho các câu khác nhau ​*S​*.
 
 > Our discussion has focused on a single example. To apply the Optimization Verification test in practice, you should examine the errors in your dev set. For each error, you would test whether Score​<sub>A​</sub>(​*S*<sup>x</sup>) > Score​<sub>A​</sub>(​*S*<sub>out</sub>). Each dev example for which this inequality holds will get A​ A​ out​ marked as an error caused by the optimization algorithm. Each example for which this does not hold (Score​<sub>A​</sub>(​S<sup>x</sup>) ≤ Score​<sub>A​</sub>(​S<sub>out</sub>)) gets counted as a mistake due to the way you’re computing Score​<sub>A​</sub>(.).
 
---> _replace THIS LINE by your translation for the above line_
+Thảo luận của chúng tôi đã tập trung vào một ví dụ duy nhất. Để áp dụng kiểm tra xác minh tối ưu hóa trong thực tế, bạn nên kiểm tra các lỗi trong tập phát triển của mình. Đối với mỗi lỗi, bạn sẽ kiểm tra xem Điểm​<sub>A​</sub>(​*S*<sup>x</sup>) > Điểm​<sub>A​</sub>(​*S*<sub>out</sub>) không. Mỗi ví dụ trong tập phát triển mà bất đẳng thức này giữ sẽ nhận được A​. A​ được đánh dấu là lỗi gây ra bởi thuật toán tối ưu hóa. Mỗi ví dụ không giữ Điểm​<sub>A​</sub>(​S<sup>x</sup>) ≤ Điểm​<sub>A​</sub>(​S<sub>out</sub>)) được tính là một lỗi do cách bạn tính toán Điểm​<sub>A​</sub>(.).
```

Comment:
> Mỗi ~~ví dụ~~ _mẫu_ trong tập phát triển mà bất đẳng thức này ~~giữ~~ _thoả mãn__ sẽ nhận được A​ ...
* "does not hold" -> "không thoả mãn"

Đoạn này bản gốc bị lỗi. Em sửa lại cả bản gốc nhé. Sửa xong cc anh để anh vào review tiếp.

Reply:
@tiepvupsu e vừa update lại, a Tiệp review xem ạ.